tests/installation/addon_products_sle.pm

```diff
@@ -155,22 +171,24 @@ sub run {
         }
     }
     if (get_var("ADDONURL")) {
+        test_addonurl if get_var('WORKAROUND_MODULES');
         if (match_has_tag('inst-addon')) {
             send_key 'alt-k';                                                   # install with addons
         }
         else {
             send_key 'alt-a';
         }
-        for my $addon (split(/,/, get_var('ADDONURL'))) {
+        for my $addon (split(/,/, get_required_var('ADDONURL'))) {
```

Comment:
`get_required_var` is not really required here, as on L#174 we already checked that it's set.